erp-client/src/styles.scss

```diff
@@ -1,2 +1,69 @@
-/* You can add global styles to this file, and also import other style files */
-@import '@angular/material/prebuilt-themes/indigo-pink.css';
\ No newline at end of file
+// Angular Material Core Theme
+@import '~@angular/material/theming';
+
+// Include non-theme styles for core.
+@include mat-core();
+
+// Define your application's custom theme.
+// TODO: Determine what actual colors we want to use.
+$primary: mat-palette($mat-indigo);
+$accent:  mat-palette($mat-pink, A200, A100, A400);
+$theme: mat-light-theme($primary, $accent);
```

Comment:
I abstracted this away from a 100% prebuilt theme so that whichever theme colors we choose can be referenced in other stylesheets (I needed to reference the primary theme color in [the createAward.scss stylesheet](https://github.com/taylor-jones/employee-recognition-portal/blob/jones/create-award-component/erp-client/src/app/components/award/createAward.component.scss).

This also gives us a bit more control over the theming. I don't have any preference as to what colors we go with. I wonder if maybe we could pick a combo that would look nice w/ Tucker's logo colors?

Reply:
Sounds good. I only added the prebuilt theme for quickness and so my components wouldn't suck but figured we'd change as we went on. Good with using whatever we decide is best.